### PR TITLE
More Python 3.6

### PR DESCRIPTION
**Pull request by cdeil**
This PR adds Python version 3.6 for
* `py-pywavelets`
* `py-pluggy`
* `py-tox`

@tobypeterson or anyone - OK?

### CODE REVIEW

File: python/py-tox/files/tox36 [hunk 1]
```diff
@@ -0,0 +1 @@
+bin/tox-3.6
```
Comment (l2dy) [inline python/py-tox/files/tox36]: What is this for?

Comment (l2dy) [inline python/py-tox/files/tox36]: What is this for?

Comment (cdeil) [inline python/py-tox/files/tox36]: I think this stub file is needed to make `port select` work for `tox`:
```
$ port info tox_select
tox_select @0.1 (python)

Description:          This port installs files that allow 'port select' to be used to create links to the preferred default version of tox.
Homepage:             http://www.macports.org/

Platforms:            darwin
License:              MIT
Maintainers:          none
```

I just copied the existing `python/py-tox/files/tox35` to `python/py-tox/files/tox36` and changed the content from `bin/tox-3.5` to `bin/tox-3.6`.

It looks like maybe there's a tox-quickstart line missing in that file?
```
$ sudo port select --set tox tox36
Selecting 'tox36' for 'tox' succeeded. 'tox36' is now active.
$ tox<TAB>
tox                 tox-3.6             tox-quickstart-3.6  
```

Is there a list of Github usernames for port maintainers somewhere?
Then we could simply @mention the tox maintainer here and ask.

Comment (l2dy) [inline python/py-tox/files/tox36]: @steenzout

**Pull request merged**